zen32_exhaust_fan.yaml

```diff
@@ -14,10 +14,14 @@ blueprint:
         The Zooz Zen32 Scene Controller that will controll the fan's state
       selector:
         device:
-          integration: zwave_js
-          manufacturer: Zooz
-          model: ZEN32
-          multiple: false
+          filter:
+            - integration: zwave_js
+              manufacturer: Zooz
+              model: ZEN32
+            - integration: zwave_js
+              manufacturer: Zooz
+              model: ZEN32 800LR
+            multiple: false
```

Comment:
I'm pretty sure the issue was that `multiple` is now indented too far.  I think it just needs to be at the same indentation level as `filter`.